bin/migration/tables/portalaccess.py

```diff
@@ -0,0 +1,55 @@
+from .helpers import check_existing_record, parse_to_timestamp, audit_entry_creation
+from datetime import datetime
+import uuid
+
+class PortalAccessManager:
+    def __init__(self, source_cursor):
+        self.source_cursor = source_cursor
+
+    def get_data(self):
+        self.source_cursor.execute("SELECT * FROM public.users WHERE prerole = 'Level 3' AND (sendinvite ILIKE 'true' OR invited ILIKE 'true')")
+        return self.source_cursor.fetchall()
+
+    def migrate_data(self, destination_cursor, source_user_data):
+        batch_portal_user_data = []
+
+        for user in source_user_data:
+            user_id = user[0]
+
+            if not check_existing_record(destination_cursor,'portal_access', 'user_id', user_id):
+                id=str(uuid.uuid4())
+                password = 'password' # ?
+                status = 'active' # ? 
```

Comment:
Not sure if you saw this: https://github.com/hmcts/pre-api/pull/62#discussion_r1410773239. We had to change all of the enums in the schema to be capitalised.
```suggestion
                status = 'ACTIVE' # ? 
```